docs/api.md

```diff
@@ -127,28 +127,16 @@ Below is a table of all base permissions.
 
 | Code              | Description                                              |
 | ----------------- | -------------------------------------------------------- |
-| manageServer      | Allows changes to [server settings](#settings).          |
-| manageUsers       | Allows for updating of users other than yourself, and allows deletion of users. |
-| manageRoles       | Allows creation/deletion/modification of [roles](#roles). |
-| manageChannels    | Allows management and editing of [channels](#channels) and their permissions. |
-| manageEmotes      | Allows for creation and removal of [emotes](#emotes).    |
-| uploadImages      | Allows [image uploads](#upload-image).                   |
-| allowNonUnique    | Allows the creation of things with non-unique [names](#names). |
-
-</details>
-
-<details id='channel-permissions'><summary><b>Channel permissions</b></summary>
-
-A set of channel-specific permissions can be set for different [channels](#channels). These relate to channel-specific actions, such as being able to read messages or send messages. These are different to base permissions as they are still per-role but are also per-channel.
-
-Below is a table of all channel permissions.
-
-| Code              | Description                                              |
-| ----------------- | -------------------------------------------------------- |
-| VIEW              | Allows for viewing of the channel in the [channel list](#channel-list). |
-| READ_MESSAGES     | Allows for viewing of channel [messages](#messages).     |
-| SEND_MESSAGES     | Allows for [sending messages](#send-message).            |
-| MANAGE_PINS       | Allows [updates to channel pins](#pin).                  |
+| `manageServer`    | Allows changes to [server settings](#settings).          |
+| `manageUsers`     | Allows for updating of users other than yourself, and allows deletion of users. |
+| `manageRoles`     | Allows creation/deletion/modification of [roles](#roles). |
+| `manageChannels`  | Allows management and editing of [channels](#channels) and their permissions. |
+| `managePins`      | Allows for [pinning](#pin) and [unpinning](#unpin) of messages. |
+| `manageEmotes`    | Allows for creation and removal of [emotes](#emotes).    |
+| `readMessages`    | Allows for viewing of channel [messages](#messages); if false, the channel does not show up in the channel list. |
+| `sendMessages`    | Allows for [sending messages](#send-message).            |
+| `uploadImages`    | Allows [image uploads](#upload-image).                   |
+| `allowNonUnique`  | Allows the creation of things with non-unique [names](#names). |
```

Comment:
(While you're at it, can you make emote shortcodes just strings that don't contain `:`?)

Reply:
Sure. Could you clarify to me exactly what non-unique things `allowNonUnique` will let you make? Channels and roles, I assume, but anything else?